lambdas/build-cri-oauth-request/src/test/java/uk/gov/di/ipv/core/buildcrioauthrequest/BuildCriOauthRequestHandlerTest.java

```diff
@@ -1325,6 +1326,83 @@ void shouldOnlyEmailForF2FAndAllowCRIConfiguredSharedClaimAttr() throws Exceptio
         verify(mockIpvSessionService, times(1)).updateIpvSession(any());
     }
 
+    @Test
+    void shouldOnlyEmailForF2FAndAllowCRIConfiguredSharedClaimAttrWithCorrectStrengthScore()
```

Comment:
Might be cleaner if this exclusively tests the evidence request  rather than everything else and rename it something like `shouldSetEvidenceRequestForF2FWithMinStrengthScoreForP1` ?